Documentation/ProjectFile/prj/process_variables/process_variable/source_terms/source_term/Volumetric/t_parameter.md

```diff
@@ -0,0 +1,2 @@
+The name of the parameter that defines value that should be used for the source
+term.
```

Comment:
`the value`.

Maybe a comment that nodal parameters are not supported would be appropriate. And maybe there should also be an addendum that the user should carefully check which physical quantity that parameter is. It's not necessarily the same as `\dot{primary_variable}`.

Reply:
I do not understand what is meant by 'nodal parameter'. Can you explain, please?

Reply:
Suggestion to keep the comment as is, and bake another PR handling this case.